data-processing/data_manipulation/llm_api_service/qa_provider_zhi_pu_ai_online.py

```diff
@@ -73,14 +72,28 @@ def generate_qa_list(
                     f"{log_tag_const.ZHI_PU_AI} Cannot access the ZhiPuAI service.\n",
                     f"The error is: \n{response['msg']}\n"
                 ]))
+                return {
+                    'status': 1000,
+                    'message': traceback.format_exc(),
+                    'data': result
+                }
         except Exception as ex:
             result = []
             logger.error(''.join([
                 f"{log_tag_const.ZHI_PU_AI} Cannot access the ZhiPuAI service.\n",
                 f"The tracing error is: \n{traceback.format_exc()}\n"
             ]))
+            return {
+                'status': 1001,
```

Comment:
1001, is this a http code or some customized error code?

Reply:
customized error code